src/components/OrgFile/components/AttributedString/index.js

```diff
@@ -20,12 +20,19 @@ export default ({ parts, subPartDataAndHandlers }) => {
           case 'link':
             const uri = part.getIn(['contents', 'uri']);
             const title = part.getIn(['contents', 'title']) || uri;
-
-            return (
-              <a key={part.get('id')} href={uri} target="_blank" rel="noopener noreferrer">
-                {title}
-              </a>
-            );
+            if (uri.startsWith("file:")) {
+              return (
+                <a key={part.get('id')} href={uri.substr(5)} rel="noopener noreferrer">
+                  {title}
+                </a>
+              );
+            } else {
+              return (
+                <a key={part.get('id')} href={uri} target="_blank" rel="noopener noreferrer">
```

Comment:
I'm pretty sure there is a way to simplify this. I think there must be a syntax to make rendering an HTML attribute depend on a bool `isFileLink`. And with a `uri = isFileLink ? uri.substr(5) : uri` or something we could avoid the duplication of the three lines of `<a>`.

Reply:
Yes, that sounds like a good refactor.

Reply:
OK sure.  I didn't go for that initially because I didn't know how to do that whilst avoiding an empty `target=""` or even `target` attribute due to `target={isFileLink ? "" : "_blank"}`, but I guess that tiny cosmetic issue hidden in the HTML source is still preferable to duplicated code.